sdk/communication/Azure.Communication.Calling.Server/src/swagger/Sample-Swagger.json

```diff
@@ -1033,6 +1099,10 @@
                 "audioFileId": {
                     "description": "An id for the media in the AudioFileUri, using which we cache the media resource.",
                     "type": "string"
+                },
+                "callbackUri": {
+                    "description": "The callback URI.",
```

Comment:
Can you add more description here. What is the purpose of the call back.

Reply:
Sure I will be more descriptive. That is the call back uri where we send back the PlayAudio Completed notification when the audio is played successfully and in case of failure a failure notification.

Reply:
Updated.